webpack.renderer.config.js

```diff
@@ -8,15 +8,24 @@ const fs = require(`fs-extra`)
 
 const HtmlWebpackPlugin = require(`html-webpack-plugin`)
 const VueLoaderPlugin = require(`vue-loader/lib/plugin`)
+const BundleAnalyzerPlugin = require(`webpack-bundle-analyzer`)
+  .BundleAnalyzerPlugin
+const CleanWebpackPlugin = require(`clean-webpack-plugin`)
 
 function resolve(dir) {
   return path.join(__dirname, dir)
 }
 
+const buildPath = path.join(__dirname, `app/dist`)
+
+const devPlugins = process.env.CIRCLECI
+  ? []
+  : [new CleanWebpackPlugin([buildPath]), new BundleAnalyzerPlugin()]
+
 const rendererConfig = {
   devtool:
     process.env.NODE_ENV === `production`
-      ? `#source-map`
+      ? `#cheap-source-map`
```

Comment:
what does this mean?

Reply:
`transformed code (lines only)`
I am not sure why we shouldn't just go with full source-maps?!

Reply:
the bundle size will increase, that's basically a version that grants similar results with half space.
- source maps is the whole codebase, so the slowest option both to create and to download, usually is advised not to put it on production albeit is one of the options, in our case since we are open source this won't impact security
- cheap source maps are a representation both tree-shacked and transpiled, build time decrease a lot and to download them is a bit quicker too

- nosources-source-map would be perfect (but is slow) for Sentry only, to report lines and files, but the code is not on the map at all, but is as slow as source-map and the size doesn't improve much too

Reply:
:+1: let's it that way then. I wasn't aware that the source-maps increase the bundle size :X